specification/app/resource-manager/Microsoft.App/stable/2022-10-01/Diagnostics.json

```diff
@@ -432,6 +480,224 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{environmentName}/detectorProperties/daprApi/": {
```

Comment:
>daprApi [](http://example.com/codeflow?start=153&length=7)

This does not follow ARM api convention. Maybe /daprApi/components and /daprApi/components/{componentName}?